packages/UsageAnalysis/files/Test Track/Bio/Composition Analysis.md

```diff
@@ -0,0 +1,9 @@
+1. Open sample_FASTA.csv (sample_HELM.csv, sample_MSA.csv).
```

Comment:
Since we can now open datasets directly from test track, should we not remove this step?

Reply:
I'll do it a bit later for all scenarios